src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/If_AllowsSideEffects.txt

```diff
@@ -45,3 +45,9 @@ Blank()
 
 >> If(Value("1")>Value("0"), Blank(); 1/0; Blank(), 3)
 Error({Kind:ErrorKind.Div0})
```

Comment:
We should have similar tests where Switch or IfError has incompatible results, but things still work. For example:
- `With({a:1},Switch(a, 1, { prop:"complex" }, "scalar"))`
- `IfError({a:1}, false)`

And more complex expressions with those